### DISCUSSION TITLE
现有项目升级到umi3.5后start报错

### DISCUSSION THREAD

**Discussion by mengxz0203** [umijs/umi#10620] [Q&A]
有没有好心人帮忙解答下是为啥呢
![image](https://user-images.githubusercontent.com/25950583/221503995-93343e76-73a7-4f7d-b403-e9033dce44f3.png)
![image](https://user-images.githubusercontent.com/25950583/221504064-2f2f14d4-017a-4b83-bae9-48020907c37a.png)

**Comment by fz6m** [comment]
尝试关闭 `mfsu` ，不使用 `webpack5` ，这个报错是因为你配置的某些选项在 webpack 5 里不符合要求（可以尝试删除某些 webpack 配置再看看）。
